zcutil/genhushconfig

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Generate a new Hush node config
+
+DIR="$HOME/.hush"
+if [ -d "$DIR" ]; then
```

Comment:
@madbuda I was thinking about that. What edge case are we trying to detect? An existing but empty .hush/ ? Or a .hush/ dir with NO hush.conf (slightly different). If you tell me the edge cases to detect and what should be done, I can try my best. I am not sure what to do if .hush/ exists but hush.conf does not. You need to check if every single possible file/dir exists, to detect a partial/corrupt copied .hush/ directory. Maybe just check for hush.conf and wallet.dat ?